WORKING_REMOTE.md

```diff
@@ -0,0 +1,93 @@
+# Working Remote
+
+*The goal of this document is to improve collaboration between people who
+ are not working in the same physical location.*
+
+ **Important: This document is not only for those who are working remotely,
+ but for everyone working at Hyper.
+ Successfully collaboration requires effort from everyone.**
+
+We are all used to speaking with people who are not in the same
+physical place as us. Although we communicate a bitt less effectively over
+phone than when do in person it's usually not a big problem.
+
+However this communication challenge becomes much more apparent to us when
+we're trying to solve complex problems together.
+
+There are many things that makes communicating with remote people more
+difficult. Let's call them communication barriers.
```

Comment:
Change difficult to challenging. It is not necessarily more difficult, however the challenges are different from those we a re used to. Also our usual communication tool belt is not as well equipped to working remotely as working in the same physical location,